.github/workflows/build.yml

```diff
@@ -42,22 +42,17 @@ jobs:
         env:
           CC: ${{ matrix.compiler }}
         run: |
-          # Configure
+          # Base flags
           export CFLAGS="-Wall -Werror"
           export LDFLAGS="-pthread -lpthread"
```

Comment:
```
meson setup -Dwerror=true
```

The -Wall is covered by meson.build default options for warning_level

I don't get the LDFLAGS here? The meson.build has `dependency('threads')` and I would presume an autotools build should be adding that in configure.ac or Makefile.am, not in the LDFLAGS...

Reply:
And sure enough, AX_PTHREAD / PTHREAD_LIBS / PTHREAD_CFLAGS is present in the autotools build, so this was probably redundant all along?